egs2/fisher_callhome_spanish/st1/local/data.sh

```diff
@@ -75,15 +75,9 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
         data/train \
         data/fisher_train data/callhome_train/ 
 
-    # temporarally using first reference as target text
-    for dataset in fisher_dev fisher_dev2; do
```

Comment:
We will have a check for `text.${tgt_case}.en` at the end, so it's necessary to have the code here

Reply:
I did not understand this comment. Can you explain?

Reply:
We force to have utt_extra_files at `https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/st1/st.sh#L297` This file will be checked in following scripts at https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/st1/st.sh#L503 , https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/st1/st.sh#L549 and many others, so it's necessary to have include that in the test sets.